packages/dialtone-vue2/components/emoji_text_wrapper/emoji_text_wrapper.vue

```diff
@@ -49,16 +49,21 @@ export default {
      * @returns {Array<VNode|string>}
      */
     replaceDtEmojis (replaceList, textContent) {
+      if (!replaceList?.length) return textContent;
```

Comment:
huh, we always pass an array, I guess we could remove the optional chaining operator

Reply:
Sure, rememeber adding that due to an error I got but I fixed that code so it's not needed right now.